scripts/run-integration-tests.sh

```diff
@@ -210,50 +210,17 @@ grep -r -q $TEST_IMAGE_VERSION $TEST_CONFIG_PATH
 sed -i'.bak' "s,602401143452.dkr.ecr.us-west-2.amazonaws.com/amazon-k8s-cni-init,$INIT_IMAGE_NAME," "$TEST_CONFIG_PATH"
 grep -r -q $INIT_IMAGE_NAME $TEST_CONFIG_PATH
 
-
 if [[ $RUN_KOPS_TEST == true ]]; then
     export KUBECONFIG=~/.kube/config
-fi
-
-if [[ $RUN_KOPS_TEST == true ]]; then
     run_kops_conformance
 fi
-ADDONS_CNI_IMAGE=$($KUBECTL_PATH describe daemonset aws-node -n kube-system | grep Image | cut -d ":" -f 2-3 | tr -d '[:space:]')
-
-echo "*******************************************************************************"
-echo "Running integration tests on default CNI version, $ADDONS_CNI_IMAGE"
```

Comment:
The nightly & integration tests are run using the same script, so removing it here will exclude the tests against default CNI in all cases. 
We should skip this conditionally, ie, for weekly tests only.